ReSharper.FSharp/src/FSharp.Psi.Intentions/src/Intentions/UseNamedAccessAction.fs

```diff
@@ -0,0 +1,103 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Intentions
+
+open FSharp.Compiler.Symbols
+open JetBrains.ReSharper.Feature.Services.ContextActions
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Psi.ExtensionsAPI.Tree
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
+open type JetBrains.Diagnostics.Assertion
+
+[<ContextAction(Name = "UseNamedAccess", Group = "F#", Description = "Use named access inside a DU pattern")>]
+type UseNamedAccessAction(dataProvider: FSharpContextActionDataProvider) =
+    inherit FSharpContextActionBase(dataProvider)
+    
+    let mutable names = Array.empty
+    let mutable tuplePat : ITuplePat = null
+
+    override x.Text = "Use named access"
```

Comment:
It should maybe somehow include the union case name. It could help when there are nested patterns.
Or it should use the terminology from the language docs, which says 'named fields'.

It's difficult to combine these two things. 🙂

Reply:
Maybe something like this (for now), even though it's not the prettiest title I've suggested: `"Use named fields for {unionCase.Name}"`

Reply:
Done!